upi/azure/04_bootstrap.json

```diff
@@ -50,7 +50,8 @@
     "identityName" : "[concat(parameters('baseName'), '-identity')]",
     "vmName" : "[concat(parameters('baseName'), '-bootstrap')]",
     "nicName" : "[concat(variables('vmName'), '-nic')]",
-    "imageName" : "[concat(parameters('baseName'), '-image')]",
+    "galleryName": "[concat('gallery_', replace(parameters('baseName'), '-', '_'))]",
+    "imageName" : "[parameters('baseName')]",
```

Comment:
when try to create arm cluster, 
`"The selected VM size 'Standard_D4ps_v5' cannot boot Hypervisor Generation '1'. `
so the bootstrap.json need 
`    "imageName" : "[concat(parameters('baseName'), if(equals(parameters('hyperVGen'), 'V2'), '-gen2', ''))]",`

Reply:
:facepalm: how did I miss that?

Reply:
after changed the 04_bootstrap.json, I can create UPI cluster on arm64.